Doc/Makefile

```diff
@@ -15,10 +15,10 @@ biopdb_faq.pdf: biopdb_faq.tex
 	pdflatex biopdb_faq.tex
 
 Tutorial.html: Tutorial.tex Tutorial/chapter_*.tex version.sh
-	hevea -exec ./version.sh -fix Tutorial.tex
```

Comment:
I see that ``xxdate.exe`` is a helper script with ``hevea`` (I was worried about the cross platform implications, should be fine).